OracleWebLogic/dockerfiles/12.2.1.1/Dockerfile.developer

```diff
@@ -63,8 +62,8 @@ RUN chmod a+xr /u01 && \
     chown oracle:oracle -R /u01 && \
     rm /u01/$FMW_JAR /u01/$FMW_PKG /u01/oraInst.loc /u01/install.file
 
-USER oracle
-WORKDIR $ORACLE_HOME 
+COPY container-scripts/* /u01/oracle/
+WORKDIR ${ORACLE_HOME} 
 
-# Define default command to start bash. 
-CMD ["bash"]
+# Define default command to start script. 
+CMD ["sh","./createAndStartEmptyDomain.sh"]

```

Comment:
The shebang comannd in the script points to /bin/bash but you're firing it with sh. I would suggest flagging createAndStartEmptyDomain.sh as executable and calling it via ENTRYPOINT instead of CMD.

See the **Consistency** section in the official Docker library repo for more info on this: https://github.com/docker-library/official-images/blob/master/README.md